Testing/functional/tests/cypress/integration/e2e/covid19/pcrTestKit.js

```diff
@@ -31,6 +31,7 @@ const processedBanner = "[data-testid=alreadyProcessedBanner]";
 
 describe("Authenticated PCR Test Kit Registration", () => {
     beforeEach(() => {
+        Cypress.session.clearAllSavedSessions();
```

Comment:
I suggest we clear the sessions right after actions in the tests that invalidate them (e.g. logouts), instead of at the start of tests (where they're not always needed).

Reply:
Taking the clear session out of the beforeEach and putting it in at the end of a test causes some tests to fails. Putting in the beforeEach ensures there is a new login for those tests.  As per description, there are some tests that are being disrupted by tests that have logouts and by tests that maintain data in the session.

Reply:
Moved Cypress.session.clearAllSavedSessions() to an afterEach block.  Bug AB#14444 has been created to find a more elegant solution.